watertap/unit_models/translator_adm1_asm1.py

```diff
@@ -0,0 +1,338 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Translator block representing the ADM1/ASM1 interface.
```

Comment:
Although this is formulated in a similar fashion to a unit model, I think this translator block belongs in the property_models folder. Ideally, we would have something like
`property_models/wastewater/translators/translator_adm1_asm1.py`
`property_models/wastewater/anaerobic_digestion/ADM1_files...`
`property_models/wastewater/activated_sludge/ASM_files...`

Reply:
@adam-a-a one of the only reasons I would avoid this change is because of how the translator blocks are used in the flowsheet. They are used like unit models, even having streams come in and out.

Reply:
Fair enough! I suppose we can consider any future sub-folders in a subsequent PR.